src/components/Navbar/Navbar.css

```diff
@@ -0,0 +1,53 @@
+nav {
+  padding: 1rem 4rem;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.bookstore-title {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.nav-links {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  display: flex;
+}
+
+.books-link {
+  width: 3.688rem;
+  height: 1rem;
+  text-decoration: none;
+}
```

Comment:
- [ ] Please change nav link color to black and move them to the left as given in the design

**Screenshot**
![image](https://user-images.githubusercontent.com/69011963/173164971-175a28b6-5420-4b52-82b5-09e761327c1d.png)